docs/docs/adrs/adr-005-cryptographic-equivocation-verification.md

```diff
@@ -16,27 +17,83 @@ Proposed
+
 - We cannot derive an infraction height from the evidence, so it is only possible to tombstone validators, not actually slash them.
-  To explain the technical reasons behind this limitation, let's recap the initial [consumer initiated slashing logic](https://github.com/cosmos/ibc/blob/2aa283d4bc8b747cbb860c0f94389d7a62bb802b/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md#consumer-initiated-slashing). 
-  In a nutshell, consumer heights are mapped to provider heights through `VSCPackets`, namely through the so called `vscIDs`. 
-  When an infraction occurs on the consumer, a `SlashPacket` containing the `vscID` obtained from mapping the consumer infraction height is sent to the provider. 
-  Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height, which is used to slash the misbehaving validator. 
-  In the context of untrusted consumer chains, all their states, including `vscIDs`, could be corrupted and therefore cannot be used for slashing purposes.
+To explain the technical reasons behind this limitation, let's recap the initial consumer initiated slashing logic.
+In a nutshell, consumer heights are mapped to provider heights through VSCPackets, namely through the so called vscIDs.
+When an infraction occurs on the consumer, a SlashPacket containing the vscID obtained from mapping the consumer infraction height
+is sent to the provider. Upon receiving the packet, the provider maps the consumer infraction height to a local infraction height,
+which is used to slash the misbehaving validator. In the context of untrusted consumer chains, all their states, including vscIDs,
+could be corrupted and the ore cannot be used for slashing purposes.
+
+- Currently, the endpoint can only handle "equivocation" light client attacks. This is because the "lunatic" attacks require the endpoint to possess the ability to dissociate which header is conflicted or trusted upon receiving a misbehavior message. Without this information, it's not possible to define the Byzantine validators from the conflicting headers (see [comment](https://github.com/cosmos/interchain-security/pull/826#discussion_r1268668684)).
 
 
 ## Consequences
 
-After this ADR is applied, it will be possible for the provider chain to tombstone validators who committed a light client attack.
+### Positive
+- After this ADR is applied, it will be possible for the provider chain to tombstone validators who committed a light client attack.
+
+### Negative
+- The misbehaviour verification process requires to copy a lot of unexposed IBC methods.
```

Comment:
Is this still true? I thought we can use [CheckMisbehaviourAndUpdateState](https://github.com/cosmos/ibc-go/blob/8f53c21361f9d65448a850c2eafcf3ab3c384a61/modules/light-clients/07-tendermint/types/misbehaviour_handle.go#L16C4-L16C35).

Reply:
Good point. 

[CheckMisbehaviourAndUpdateState](https://github.com/cosmos/ibc-go/blob/8f53c21361f9d65448a850c2eafcf3ab3c384a61/modules/light-clients/07-tendermint/types/misbehaviour_handle.go#L16C4-L16C35) accepts time violations as valid misbehaivour, so I was initially thinking to rewrite it and copy all the unexposed methods.

Instead, I will just add the opposite of this [condition](https://github.com/cosmos/ibc-go/blob/8f53c21361f9d65448a850c2eafcf3ab3c384a61/modules/light-clients/07-tendermint/types/misbehaviour_handle.go#L54-L58) before calling [CheckMisbehaviourAndUpdateState](https://github.com/cosmos/ibc-go/blob/8f53c21361f9d65448a850c2eafcf3ab3c384a61/modules/light-clients/07-tendermint/types/misbehaviour_handle.go#L16C4-L16C35).